ios/RnSquareMobilePayments.swift

```diff
@@ -117,7 +117,7 @@ class SquareMobilePayments: RCTEventEmitter {
 
     let currency = Currency(curr)
     let amountMoney = Money(amount: UInt(amount), currency: currency)
-    var params = PaymentParameters(idempotencyKey: UUID().uuidString, amountMoney: amountMoney);
+    var params = PaymentParameters(idempotencyKey: UUID().uuidString, amountMoney: amountMoney, processingMode: ProcessingMode.onlineOnly);
```

Comment:
Hm, so this should definitely be passed into the function instead of hardcoded, but like we discussed, we are moving away from this package in favor of the official package anyway, so I'm fine with this.